src/thunderbots/software/ai/hl/stp/tactic/shoot_goal_tactic.cpp

```diff
@@ -45,19 +52,19 @@ double ShootGoalTactic::calculateRobotCost(const Robot &robot, const World &worl
         // If we can intercept the ball, use the distance to the intercept point.
         // We normalize with the total field length so that robots that are within the
         // field have a cost less than 1
-        cost = (ball_intercept_opt->first - robot.position()).len() /
+        cost = std::pow((ball_intercept_opt->first - robot.position()).len(), 6) /
                world.field().totalLength();
```

Comment:
What's up with the `pow` here and below?

Reply:
This was a "hack" to increase the priority of this tactic getting assigned. I'm fine removing it unless we want to be really strict about putting RoboCup code in this PR, and fixing it separately.

Reply:
Yeah, I'd say lets remove it here.

Reply:
Ok, can we file a ticket to properly handle goalies? Let's chat quickly in person about this one.

Reply:
Removed the power hack and proper goalie handling is on the task list.